app/src/main/res/layout/activity_main.xml

```diff
@@ -5,15 +5,15 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@drawable/wallpaper_main_background"
-    tools:context=".ParkingActivity">
+    tools:context=".activities.ParkingActivity">
 
     <TextView
         android:id="@+id/text_main_parking_title"
         style="@style/TextMainStyle"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:text="@string/text_main_activity_parking"
-        app:layout_constraintBottom_toBottomOf="parent"
+        android:layout_marginTop="@dimen/reservation_activity_margin_size_normal"
```

Comment:
mmm the name not represents the WHAT code style, rename it

Reply:
Done

Reply:
this layout is called activity_main , use that to rename

Reply:
I use that dimension in the main activity and in the reservation activity, for that reason I express it in a general way